### PR TITLE
Autocomplete2

### PR DESCRIPTION
**Pull request by cfchacon10**
Created a new branch because folder names and things broke. I honestly didn't know how to fix it but this is directly from the branch "autocomplete" so the same changes are here, but now the new service request screen page is working for mobile. A stupid space.

### CODE REVIEW

File: homepairs/homepairsUI/src/elements/Forms/GoogleInputForm.tsx [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
 import React from 'react';
-import { Text, View, TextInput, ViewStyle, StyleSheet, ScrollView } from 'react-native';
+import { AppState } from 'homepairs-types';
+import { connect } from 'react-redux';
+import { Text, View, TextInput, ViewStyle, StyleSheet, ScrollView, AsyncStorage } from 'react-native';
 import {HelperText} from 'react-native-paper';
 import {FontTheme} from 'homepairs-base-styles';
 import * as BaseStyles from 'homepairs-base-styles';
-import LocationItem from '../LocationItem/LocationItem';
+import {LocationItem} from 'homepairs-elements';
```
Comment (ItisEeron) [inline homepairs/homepairsUI/src/elements/Forms/GoogleInputForm.tsx:9]: This risks us for dependency cycles. A modules referenced in an alias should never reference the alias.

**Review by ItisEeron — DISMISSED** [review]
So I don't know what's up the google input form but it still only seems to show up when I turn CORS off. I don't know if this on the GoogleMaps api so I won't hold this one on you. We should ask Alex about it on Thursday. I doubt it on our end this time. I'm going to approve the changes but make sure to go over any comments I made.

**Review by YellowRainBoots — DISMISSED** [review]
Tested on android. Good to go! Only point i'd make is- this would be a really good hotfix, or we could maybe put a new version out at the end of this one week sprint. Only thing is, we need to update new service requests to Tommy's change / required isPm field.

File: homepairs/homepairsUI/src/elements/Forms/GoogleInputForm.tsx (hunk 1 above)
Comment (cfchacon10) [reply homepairs/homepairsUI/src/elements/Forms/GoogleInputForm.tsx:9]: Fixed

**Pull request merged**